azext_iot/central/_help.py

```diff
@@ -44,6 +57,257 @@ def load_central_help():
+            --dest-id {destintionid}
+            --type blobstorage@v1
+            --name {displayname}
+            --authorization "{
+              'type': 'connectionString',
+              'connectionString':'DefaultEndpointsProtocol=https;AccountName=[accountName];AccountKey=[key];EndpointSuffix=core.windows.net',
+              'containerName': 'test'
+            }"
+
+        - name: create a Azure Data Explorer export destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --type dataexplorer@v1
+            --name {displayname}
+            --cluster-url {clusterurl}
+            --database {database}
+            --table {table}
+            --authorization "{
+              'type': 'servicePrincipal',
+              'clientId': '3b420743-2020-44c6-9b70-cc42f945db0x',
+              'tenantId': '72f988bf-86f1-41af-91ab-2d7cd011db47',
+              'clientSecret': '[Secret]'
+            }"
+
+        - name: create an Event Hub export destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --type eventhubs@v1
+            --name {displayname}
+            --authorization "{
+              'type': 'connectionString',
+              'connectionString': 'Endpoint=sb://[hubName].servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=*****;EntityPath=entityPath1'
+            }"
+
+        - name: create an Service Bus Queue destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --type servicebusqueue@v1
+            --name {displayname}
+            --authorization "{
+              'type': 'connectionString',
+              'connectionString': 'Endpoint=sb://[namespance].servicebus.windows.net/;SharedAccessKeyName=xxx;SharedAccessKey=[key];EntityPath=[name]'
+            }"
+
+        - name: create an Service Bus Topic destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destintionid}
+            --type servicebustopic@v1
+            --name {displayname}
+            --authorization "{
+              'type': 'connectionString',
+              'connectionString': 'Endpoint=sb://[namespace].servicebus.windows.net/;SharedAccessKeyName=xxx;SharedAccessKey=[key];EntityPath=[name]'
+            }"
+    """
+
+    helps[
+        "iot central export destination update"
+    ] = """
+        type: command
+        short-summary: Update an export destination for an IoT Central application.
+        long-summary: Destination type is not changable once it created, you need to create a new destintaion with new type.
```

Comment:
sp/grammar  `The destination type is immutable once it is created. A new destination must be created with the new type.`

Reply:
fixed